### DISCUSSION TITLE
Configuation as env vars

### DISCUSSION THREAD

**Discussion by timheuer** [milvus-io/milvus#33032] [Q&A and General discussion] [answered]
I'm curious if the configuration values set in milvus.yml can be set as environment variables on the container images (standalone) using some naming convention? `section__subsection__??`

I'm looking to enable authentication and set the default root password to a different value using this method. Is that possible?  I'm not using compose and want to configure a different way.

**Comment by yhmo** [comment, marked as answer]
The default root user and password is root:Milvus, not configurable. You can use python sdk to change its password: 
`utility.update_password(user="root", old_password="Milvus", new_password="xxxxxx")`

**Reply by timheuer** [reply]
Thanks, is there no means to set this before startup? Trying in a managed k8s environment to use env to communicate between and not have a startup service need a client to call update_password if can be avoided.

**Reply by yhmo** [reply]
From the source code I didn't see any possibilities to config the username/password for root users.

**Reply by timheuer** [reply]
I logged https://github.com/milvus-io/milvus/issues/33058 for consideration.
